FWCore/Services/plugins/CPU.cc

```diff
@@ -16,6 +16,20 @@
 #include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
 #include "FWCore/Utilities/interface/CPUServiceBase.h"
 
+#include "cpu_features/cpu_features_macros.h"
+
+#if defined(CPU_FEATURES_ARCH_X86)
+#include "cpu_features/cpuinfo_x86.h"
+#elif defined(CPU_FEATURES_ARCH_ARM)
+#include "cpu_features/cpuinfo_arm.h"
+#elif defined(CPU_FEATURES_ARCH_AARCH64)
+#include "cpu_features/cpuinfo_aarch64.h"
+#elif defined(CPU_FEATURES_ARCH_MIPS)
+//#include "cpu_features/cpuinfo_mips.h"
```

Comment:
Is this commented out line wanted? (Just checking because in the test [here below](https://github.com/cms-sw/cmssw/pull/40050/files#diff-26cf0a5ca9068ad1c3f63c410e9fd7050b82bd6c0ec079e8106cc7544a793d60R14) it is maintained)

Reply:
Or should we just remove the MIPS branch altogether (from here and the test)? I'd think it to be very unlikely for us to start using MIPS. RISC-V would be more interesting, but I see support for RISC-V is still in an open PR.